### PR TITLE
added max and min age on class middleware

### PR DESCRIPTION
**Pull request by LeonelHays**
## Description

Trello card: https://trello.com/c/hK3T89yR/167-update-classes-table-to-include-new-inputs

I added the max and min inputs for the class middleware because the stakeholder gave us an idea of how it is going to work. Both input fields are integers and are null-able since it is just a recommendation.
(We may need to update the seeds in the future.)

## Loom Video

https://www.loom.com/share/b51640d20f4a4f96966ef121c971f3a4

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by CamMeiners — APPROVED** [review]
The code looks good. Thinking about edge case scenarios, can we make it so the max_age integer cannot be less than the min_age?

File: README.md [hunk 1]
```diff
@@ -163,6 +163,24 @@ Updated table Source: https://dbdesigner.page.link/WTZRbVeTR7EzLvs86
...
+  open_seats_remaining: INTEGER,
+  intructor_id: INTEGER,
+  course_type_id: INTEGER,
+  start_time: TIME,
+  end_time: TIME,
+  start_date: DATE,
+  end_date: DATE,
+  location: STRING,
+}
+
```
Comment (RoseGabriely) [inline README.md:183]: Thanks for adding this to the documentation!

File: data/migrations/20211102125418_courses.js [hunk 2]
```diff
@@ -10,6 +10,9 @@ exports.up = (knex) => {
     .createTable('classes', (table) => {
       table.increments('class_id');
       table.integer('size').notNullable();
+      table.integer('min_age');
+      table.integer('max_age');
+      table.integer('sessions');
       table.integer('open_seats_remaining');
```
Comment (RoseGabriely) [inline data/migrations/20211102125418_courses.js:16]: could you please add a day field (days of the week), and a course_type_id that references the course_types table

**Review by RoseGabriely — CHANGES_REQUESTED** [review]
Great start, we also need to go back and make sure that anything that depended on the previous code has been updated. This will include all of the seeds, models, and middleware that were using this table.

**Review by 19jlevitre — COMMENTED** [review]
Changes look good and code looks clean. Very thoughtful to add such good docs to the ReadMe. Seems like Rose is right about adding the days available for classes, but that seems like itd be it

**Review by mcmillanj — APPROVED** [review]
you fix the changes that were reguested plus the min and max  age  added to table .great job

**Pull request closed without merging**